modules/tea.py

```diff
@@ -0,0 +1,25 @@
+from .base import Module
+
+class Tea(Module):
+    DESCRIPTION = 'Spills the tea.'
+    def response(self, query, message):
+        msg1 = '{0: <13}'.format(query[0:13])
+        msg2 = '{0: <11}'.format(query[13:24])
+        msg3 = '{0: <9}'.format(query[24:33])
```

Comment:
I have no idea how this formatting works but you probably know what you're doing